Winget-AutoUpdate-Install.ps1

```diff
@@ -176,13 +183,19 @@ function Install-WingetAutoUpdate {
             New-Item -ItemType Directory -Force -Path $WingetUpdatePath | Out-Null
         }
         else {
-            Remove-Item -Path "$WingetUpdatePath\*" -Exclude *.log -Recurse -Force
+            if (!$NoClean) {
+                Remove-Item "$WingetUpdatePath\*" -Force -Recurse -Exclude "*.log"
+            }
+            else {
+                #Keep critical files
+                Get-ChildItem -Path $WingetUpdatePath -Exclude included_apps.txt,mods,logs | Remove-Item -Recurse -Force

```

Comment:
we should also keep the excluded_apps.txt no ?

Reply:
Yup, that was for a loop of computers.
Think I'm done.